GitLink.sublime-commands

```diff
@@ -0,0 +1,48 @@
+[
+    {
+        "caption": "GitLink: Copy Git URL to file",
+        "command": "gitlink",
+        "args": {
+            "line": false
+        }
+    },
```

Comment:
`sublime.set_clipboard()` doesn't appear to be modifying my clipboard. Is that a know issue? I can open a new ticket and troubleshoot later

Reply:
To be honest, I'm not exactly sure.

Reply:
Found the problem, the script was failing here: https://github.com/rscherf/GitLink/pull/24/commits/e3daa8aa2ecc86d33506af02395e642ea21322a9